tasks/configure.yml

```diff
@@ -13,7 +13,7 @@
 
 - name: Download OpenVPN server keys
   become: yes
-  command: "aws s3 cp {{ openvpn.aws_s3_options }} {{ openvpn.aws_s3_path }}/{{ item }} {{ openvpn.keydir }}"
+  command: "aws s3 cp --region {{ openvpn.aws_region }} {{ openvpn.aws_s3_options }} {{ openvpn.aws_s3_path }}/{{ item }} {{ openvpn.keydir }}"
```

Comment:
Yep you can do that, it's already in the defaults for this role, this change just adds the region to the S3 download.